bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -72,6 +72,24 @@
     private int timeoutMs = BIGTABLE_CHANNEL_TIMEOUT_MS_DEFAULT;
     private int dataChannelCount = BIGTABLE_DATA_CHANNEL_COUNT_DEFAULT;
 
+    public Builder() {
+    }
+
+    private Builder(BigtableOptions original) {
+      this.projectId = original.projectId;
+      this.zoneId = original.zoneId;
+      this.clusterId = original.clusterId;
+      this.userAgent = original.userAgent;
+      this.dataHost = original.dataHost;
+      this.tableAdminHost = original.tableAdminHost;
+      this.clusterAdminHost = original.clusterAdminHost;
+      this.port = original.port;
+      this.overrideIp = original.overrideIp;
+      this.credentialOptions = original.credentialOptions;
+      this.retryOptions = original.retryOptions;
+      this.timeoutMs = original.timeoutMs;
+    }
```

Comment:
Good catch.  I added it.  I also noticed a discrepancy between the builder and the BigtableOptions naming convention (dataChannelCount vs. channelCount).